### PR TITLE
Added Moment operation at

### LINKED ISSUE #3145: Moment should have `operation_at`
There is a method called this on `Circuit`. It would make sense to live on `Moment` and then the circuit method could dispatch to it

### PR DESCRIPTION
**Pull request by exAClior**
### Purpose of PR

Added the function `operation_at` per #3145 

### Basic Code Logic

First tests if the qubit is in fact operated on for the moment. If not, return None. 
If it is return the value by moment[q] for moment being the moment we are working on and q being the qubit we want to know about. 

Fixes #3145.

### CODE REVIEW

File: cirq/ops/moment.py [hunk 1]
```diff
@@ -97,6 +97,21 @@ def operates_on(self, qubits: Iterable['cirq.Qid']) -> bool:
         """
         return bool(set(qubits) & self.qubits)
 
+    def operation_at(self, qubit: raw_types.Qid) -> Optional['cirq.Operation']:
+        """Returns the operation on a certain qubit for the moment
+
+        Args:
+            qubit: The qubit on which the returned Operation operates
+            on.
```
Comment (balopat) [inline cirq/ops/moment.py:105]: indent 4 spaces
```suggestion
                on.
```

File: cirq/ops/moment.py [hunk 2]
```diff
@@ -97,6 +97,21 @@ def operates_on(self, qubits: Iterable['cirq.Qid']) -> bool:
         """
         return bool(set(qubits) & self.qubits)
 
+    def operation_at(self, qubit: raw_types.Qid) -> Optional['cirq.Operation']:
+        """Returns the operation on a certain qubit for the moment
```
Comment (balopat) [inline cirq/ops/moment.py:101]:
```suggestion
        """Returns the operation on a certain qubit for the moment.
```
period

**Review by balopat — APPROVED** [review]
LGTM, thank you! A couple of nits though before merging

**Comment by exAClior** [comment]
> LGTM, thank you! A couple of nits though before merging

Thank you for pointing them out! Need to watch out for those things next time

**Comment by balopat** [comment]
Fixes #3339.

**Pull request merged**